cmd/logger/message/audit/entry.go

```diff
@@ -34,13 +34,13 @@ type Entry struct {
 	DeploymentID string `json:"deploymentid,omitempty"`
 	Time         string `json:"time"`
 	API          struct {
-		Name            string `json:"name,omitempty"`
-		Bucket          string `json:"bucket,omitempty"`
-		Object          string `json:"object,omitempty"`
-		Status          string `json:"status,omitempty"`
-		StatusCode      int    `json:"statusCode,omitempty"`
-		TimeToFirstByte string `json:"timeToFirstByte,omitempty"`
-		TimeToResponse  string `json:"timeToResponse,omitempty"`
+		Name            string        `json:"name,omitempty"`
+		Bucket          string        `json:"bucket,omitempty"`
+		Object          string        `json:"object,omitempty"`
+		Status          string        `json:"status,omitempty"`
+		StatusCode      int           `json:"statusCode,omitempty"`
+		TimeToFirstByte time.Duration `json:"timeToFirstByte,omitempty"`
+		TimeToResponse  time.Duration `json:"timeToResponse,omitempty"`
```

Comment:
Changing this from a string to a number will probably break most implementations. Let's change the name and also make it clear this is nanoseconds.

```suggestion
		TimeToFirstByte time.Duration `json:"timeToFirstByteNs,omitempty"`
		TimeToResponse  time.Duration `json:"timeToResponseNs,omitempty"`
```

 Why aren't the names here snake case as is common in JSON?

`omitempty` will discard 0 values. Timer precision will make some of the values not be present. Consider if that is a good idea.

Reply:
Yes, This will be a breaking change indeed @klauspost. So i thought of keeping the same name `timeToResponse`

And yes, omitempty is not effective here incase of ns, Will remove that :+1:

Reply:
And timeToFirstByte is recorded only for GET requests, So we can still keep the omitempty for TimeToFirstByte